src/components/Main/MainView/ChannelView/ChannelSidebar/ChannelSidebar.vue

```diff
@@ -7,6 +7,7 @@
         </template>
         <template #content>
           <channel-sidebar-content
+            v-model="isViewersDetailOpen"
```

Comment:
```suggestion
            v-model:isViewersDetailOpen="isViewersDetailOpen"
```
このコンポーネントのメインの機能ではないので、名前つきのものにしたほうがわかりやすそうです 🖖 
https://vuejs.org/guide/components/events.html#multiple-v-model-bindings